libnest/src/transaction/mod.rs

```diff
@@ -0,0 +1,31 @@
+//! Provides types and functions to perform transactions.
+//!
+//! Within Nest, a transaction is any operation that could affect the host filesystem.
+//! At the moment, we distinguish the following types of transactions:
+//! - Pull
+//! - Installation
+//! - Removal
+//!
+
+mod install;
+mod pull;
+mod remove;
+
+pub use self::install::InstallTransaction;
+pub use self::pull::PullTransaction;
+pub use self::remove::RemoveTransaction;
+
+/// The kind of a transaction
```

Comment:
I'm not sure the comment is exactly accurate. It looks like the remnants of an old code.

Reply:
I see how the wording can seem inaccurate here, I will update it